### DISCUSSION TITLE
ScratchAddons for mobil

### DISCUSSION THREAD

**Discussion by maDU59** [ScratchAddons/ScratchAddons#1512] [Discussion]
I think it can be cool if ScratchAddons work on Android/IOS, it is possible? (Often, I do scratch at school on my tablet, sometime I see projects on my phone, but I can't got ScratchAddons on it...)

**Comment by mxmou** [comment]
This would be really cool, but I don't think this is something we can do. You could try installing a mobile browser that supports extensions.

**Reply by maDU59** [reply]
Ok, I've already try on chrome, Samsung internet, Google, Opéra.

**Comment by cobaltt7** [comment]
There's no mobile browser that supports chromium or firefox extensions currently. There was already #343 for this, read for more info.
________________________________

**Comment by pufferfish101007** [comment]
You can use Yandex or Kiwi for this. Download the app,then go to the chrome webstore and download the extension. It seems to work for me.
